tickerv2/ticker.go

```diff
@@ -0,0 +1,93 @@
+package tickerv2
+
+import (
+	"fmt"
+	"github.com/fahimimam/busbdChckr/businfo"
+	avbusinfo "github.com/fahimimam/busbdChckr/businfo/availableBusInformation"
+	"github.com/fahimimam/busbdChckr/businfo/models"
+	"log"
+	"strings"
+	"time"
+)
+
+const StructureTypeBus = "BUS"
+
+func RunTickerV2() {
+	source := "Dhaka"
+	source = strings.ToLower(source)
+	destination := "khulna"
+	destination = strings.ToLower(destination)
+	data := avbusinfo.RequestPld{
+		Date:          "2024-04-04",
+		Identifier:    fmt.Sprintf("%s-to-%s", source, destination),
+		StructureType: StructureTypeBus,
+	}
+	// Initial fetch
+	prevData, err := fetchBusInfo(data)
```

Comment:
Does this work? Last time I checked they have special IDs for the source and destinations...